packages/devtools-launchpad/src/components/LandingPage.css

```diff
@@ -84,14 +84,19 @@
 
 .landing-page .panel header input {
   flex: 1;
-  color: var(--theme-body-color);
+  background-color: var(--theme-body-background);
+  color: var(--theme-comment);
```

Comment:
I agree it's a bit lacking in contrast. I chose to use the same style as the search inputs in the debugger. Maybe worth changing here and in the debugger.


https://github.com/devtools-html/debugger.html/blob/master/src/components/Autocomplete.css#L55
https://github.com/devtools-html/debugger.html/blob/master/src/components/EditorSearchBar.css#L23